spec/templater_spec.rb

```diff
@@ -71,7 +62,7 @@ def expect_renders(template_parameters, expected, config: Metalware::Config.new)
         alces.index: 0
         EOF
 
-        expect_renders({}, expected, config: @config)
+        expect_renders({}, expected)
```

Comment:
Where about are we starting FakeFS? I was expecting to see `FileSystem.test` somewhere